napalm/base/canonical_map.py

```diff
@@ -99,6 +101,8 @@
     "Serial": "Se",
     "TenGigabitEthernet": "Te",
     "Tunnel": "Tu",
+    "TwoGigabitEthernet": "Tw",
+    "TwentyFiveGigE": "Twe",
```

Comment:
@bbmin7b5 Wouldn't it be better to be "Two" and "Twe" as "Tw" could be ambiguous with both "TwoGigabitEthernet" and "TwentyFiveGigE"?

Reply:
@ktbyers I went with "Tw" for the "TwoGigabitEthernet" as that is how Cisco displayed the shortened form of the interface when using "show interface status" and "show ip interface brief". I figured I'd follow that model as that is what the other key/value pairs seemed to conform to.

I hope my reasoning makes sense.